lib/device.js

```diff
@@ -184,7 +212,7 @@ class Device extends EventEmitter {
 		for (;;) {
 			try {
 				const mbox = await this.readMailbox();
-				if (mbox) {
+				if (mbox && mbox.t) {
```

Comment:
What is `mbox.t`? This is a test name for a test that completed?

Reply:
I see, it is `MailboxTypes`. Got it

Reply:
't' as in 'type'.